core/src/main/java/de/danoeh/antennapod/core/util/TextUtils.java

```diff
@@ -0,0 +1,27 @@
+package de.danoeh.antennapod.core.util;
+
+import androidx.core.text.HtmlCompat;
+
+/**
+ * TextUtils contains utility methods to handle textual content seen in the podcasting world
+ */
+public final class TextUtils {
```

Comment:
This has the same name as the Android framework class `TextUtils`. This could lead to confusion. Maybe it's better to move the method to `HtmlUtils` instead.

Reply:
@ByteHamster This is why I arrived at adding a new TextUtils class:
We just discussed that some podcasting networks broke the unspoken rule and introduced HTML. We can't be sure if there would something else soon - Markdown? RestructuredText?

This was reflected in the comment above class and the only method inside it.

That way we can let this class (and the method) inside "core" to hide this style handling and our "app" can continue to invoke `TextUtils.getNormalizedText(description)` undisturbed

Thoughts?

Reply:
Does `PodcastTextUtils` sound good? Feel free to suggest any other name/ask again to drop this effort and move this inside HtmlUtils. :+1:

Reply:
Renamed